src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/configure-control-plane-monitoring.mdx

```diff
@@ -241,6 +245,11 @@ Assuming that every component is reachable at the specified port, the following
             type: bearer
 ```
 
+In this example, the integration needs to run on the same node of each control plane component having the `hostNetwork` option set to true, since it is connecting locally to each `staticEndpoint`. 
+Therefore, `controlPlane.kind` has to be kept as `DaemonSet`, moreover, the DaemonSet needs affinity rules, nodeSelector and tolerations configured so that all instances are running on all control plane nodes you want to monitor.
```

Comment:
A small issue I see suggesting this config would be that if there is a master node where one of the components is _not_ running, the integration will fail as `staticEndpoints` are a hard fail if unreachable.

I think we should mention this caveat somewhere, but I'm not fully sure how 🤔

Reply:
Whenever [you register](https://rancher.com/docs/rancher/v2.5/en/cluster-provisioning/rke-clusters/custom-nodes/agent-options/#role-options) there are only 4 options:


PARAMETER | ENVIRONMENT VARIABLE | DESCRIPTION
-- | -- | --
--all-roles | ALL=true | Apply all roles (etcd,controlplane,worker) to the node
--etcd | ETCD=true | Apply the role etcd to the node
--controlplane | CONTROL=true | Apply the role controlplane to the node
--worker | WORKER=true | Apply the role worker to the node

The suggested configuration does not have etcd because of that and because it is way more difficult to configure. MTLS, changes in etcd config itself or proxy

Reply:
<img width="977" alt="Screen Shot 2022-04-28 at 11 14 47 AM" src="https://user-images.githubusercontent.com/43335750/165719890-2a2b3153-4173-4de1-b2aa-d6575bb429b1.png">